core/include/optee_rpc_cmd.h

```diff
@@ -349,4 +354,35 @@
 
 /* End of definition of protocol for command OPTEE_RPC_CMD_SOCKET */
 
+/*
+ * Definition of protocol for command OPTEE_RPC_CMD_SUPP_PLUGIN
+ */
+
+/*
+ * Invoke tee-supplicant's plugin.
+ *
+ * [in]     value[0].a	OPTEE_RPC_SUPP_PLUGIN_INVOKE
+ * [in]     value[0].b	uuid.d1
+ * [in]     value[0].c	uuid.d2
+ * [in]     value[1].a	uuid.d3
+ * [in]     value[1].b	uuid.d4
+ * [in]     value[1].c	cmd for plugin
+ * [in]     value[2].a	sub_cmd for plugin
+ * [out]    value[2].b  length of the outbuf (memref[3]), if out is needed.
+ * [in/out] memref[3]	buffer holding data for plugin
+ *
+ * Serialized bytes of the TEE_UUID:
```

Comment:
Might be more clear to say: "UUID serialized into octets".

This is not to be confused with the in memory representation of `TEE_UUID`.